packages/payment-integration-api/src/payment-provider-authentication/payment-provider-authentication.ts

```diff
@@ -0,0 +1,3 @@
+export interface PaymentProviderAuthentication {
+    isAuthenticated?: boolean;
```

Comment:
Is this supposed to be undefined?

Reply:
Can we please remove the undefined.

Reply:
I can set default isAuthenticated value to false. But I am wondering as it is used only in StripeUPE, would it not be confusing to someone having it set to false by default? Then somebody might think  "Should I have this set to true to have authentication?". But maybe I am wrong?